plugins/org.eclipse.fordiac.ide.fb.interpreter/src/org/eclipse/fordiac/ide/fb/interpreter/Messages.java

```diff
@@ -20,6 +20,10 @@ public class Messages extends NLS {
 	public static String RecordExecutionTraceHandler_Incorrect_Selection;
 	public static String RecordExecutionTraceHandler_Select_FB_input_event;
 	public static String CreateRuntimeTestFunctionBlockHandler_Select_Service_Model;
+	public static String SelectAdapterEventDialog_0;
```

Comment:
could you please give these variables sensible names that can help later in translation